sqlalchemy_ibmi/base.py

```diff
@@ -730,68 +754,6 @@ def set_isolation_level(self, dbapi_conn, level):
     def get_isolation_level(self, dbapi_conn):
         pass
 
-    def normalize_name(self, name):
-        return self._reflector.normalize_name(name)
-
-    def denormalize_name(self, name):
-        return self._reflector.denormalize_name(name)
```

Comment:
These seem to have been removed completely having been defined in https://github.com/IBM/sqlalchemy-ibmi/pull/34/files?utf8=%E2%9C%93&diff=unified&w=1#diff-3d4ffd35dfc8aebc13a4f5024a07dbc9L56-L69

Are they no longer needed?

Reply:
Found that the implementation they have causes errors when running tests, but the super implementation of those methods works without errors.

Reply:
Ok, sounds good. Personally would have rather seen this work done in a separate PR.